_docs_integrate/13-connector-error-codes.md

```diff
@@ -0,0 +1,79 @@
+---
+title: ErrorCodes
+permalink: /integrate/errorCodes
+---
+
+Many errors are about false user input. If you cannot regognize a connection between your interactions and the error or you think the error is incorrect please report it in the [Enmeshed Issue Tracker](https://github.com/nmshd/feedback/issues).
```

Comment:
Please find a list of Enmeshed error codes below. Most often the errors occur on invalid input or actions. If you happen to find unexpected errors while using Enmeshed or cannot deduce the reason for your error, please report it in the [Enmeshed Issue Tracker](https://github.com/nmshd/feedback/issues).